ansible/roles/mordred/defaults/main.yml

```diff
@@ -26,11 +26,13 @@ mordred_aliases_url: https://raw.githubusercontent.com/chaoss/grimoirelab-sirmor
 ##   - password: "<PASSWORD>"
 ##     overwrite_roles: true
 ##     sources_repository: "<PROJECT1_SOURCE_REPO_URL>"
+##     machine: 0
```

Comment:
Instead of machine I would use the terminology of `host`. Change commit messages too.